.circleci/config.yml

```diff
@@ -2150,6 +2159,20 @@ workflows:
             - slack
             - runtimeverification
 
+  scheduled-cannon-full-tests:
```

Comment:
@mslipper can you confirm that this actually works and that this will notify us if the test fails?